result/yield-star-spacing.js

```diff
@@ -0,0 +1,24 @@
+/*eslint yield-star-spacing: ["error", "after"]*/
+/*eslint-env es6*/
+
+function* generator() {
+  yield* other();
+}
+/*eslint yield-star-spacing: ["error", "before"]*/
+/*eslint-env es6*/
+
+function *generator() {
+  yield *other();
+}
+/*eslint yield-star-spacing: ["error", "both"]*/
+/*eslint-env es6*/
+
+function * generator() {
+  yield * other();
+}
+/*eslint yield-star-spacing: ["error", "neither"]*/
+/*eslint-env es6*/
+
+function*generator() {
```

Comment:
__[development][eslint]__
* Error - 'generator' is already defined (no-redeclare)